axiom/monitors_test.go

```diff
@@ -298,7 +298,7 @@ func TestMonitor_MarshalJSON(t *testing.T) {
 				"op": "",
 				"field": "",
 				"value": null,
-				"caseInsensitive": false,
+				"caseSensitive": false,
```

Comment:
```suggestion
				"caseSensitive": true,
```
Shouldn't this be flipped?

Reply:
No. The test payload which gets encoded does not explicitly specify `monitor.CaseSensitive = true`. The zero value of `bool` is still false.

Reply:
Didn't look at the test, sounds good 👍